release-utils/scripts/upload-source.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+if [ "$#" -lt 2 ]; then
+    echo "usage: $0 upload-sources release-version destination-version"
+    exit 1
+fi
+
+location=$(dirname $0)
+version=$1
+
+mkdir $1/
+cd $1/
+
+wget https://github.com/apache/camel-quarkus/archive/refs/tags/$1.zip -O apache-camel-quarkus-$1-src.zip
+cp ../../../target/apache-camel-quarkus-$1-sbom.json apache-camel-quarkus-$1-sbom.json
```

Comment:
I may be wrong about these, but if I look at where the last release generated the files I find them at:

```
../../../target/checkout/target/camel-quarkus-3.6.0-sbom.xml
../../../target/checkout/target/camel-quarkus-3.6.0-sbom.json
```

Reply:
Thanks for pointing that out, I didn't have the time to do a dry run so I take target as location, like camel-kamelets.

Reply:
It should be fine now.